packages/api-reference/src/components/Content/Introduction/Introduction.vue

```diff
@@ -25,12 +25,12 @@ const { state, getClientTitle, getTargetTitle } = useTemplateStore()
 <template>
   <SectionContainer>
     <Section>
-      <SectionHeader
-        :loading="!info.title"
-        tight>
-        {{ info.title }}
-      </SectionHeader>
       <SectionContent :loading="!info.description && !info.title">
+        <SectionHeader
+          :loading="!info.title"
+          tight>
+          {{ info.title }}
+        </SectionHeader>
```

Comment:
This breaks the loading skeleton.

Reply:
cam fixed it with css

The motivation is the content is confined to the column and doesnt overflow into the second column